neurom/tests/test_stats.py

```diff
@@ -63,3 +63,9 @@ def test_fit_default_is_normal():
     p1, e1 = st.fit(NORMAL, 'norm')
     nt.assert_items_equal(p0, p1)
     nt.assert_items_equal(e0, e1)
+
+def test_optimal_distribution():
```

Comment:
I think here it makes sense to test whether it will find the optimal distribution, e.g. if you pass an exponential or a uniform you get that back (assuming a large enough sample.)